packages/extension/src/inpage/inpage.model.ts

```diff
@@ -1,5 +1,7 @@
 import type { AccountInterface, Provider } from "starknet"
 
+import { Network } from "./../shared/networks"
```

Comment:
does it work like this?

```suggestion
import type { Network } from "./../shared/networks"
```